src/Snackbar/SnackbarContext.js

```diff
@@ -0,0 +1,21 @@
+import { createContext, useContext, useReducer } from "react";
+import SnackbarReducer from "./SnackbarReducer";
+
+const SnackbarContext = createContext();
+
+export const SnackbarContextProvider = (props) => {
+  const [snackbar, dispatch] = useReducer(SnackbarReducer, {
+    visible: false,
+    message: ""
+  });
+
+  return (
+    <SnackbarContext.Provider
+      value={{ snackbar: snackbar, snackbarDispatch: dispatch }}

```

Comment:
you can directly name the reducer function and variable name in line 7.....would reduce excess line...

Reply:
i just wanted to use the name dispatch in the actual reducer since this is something everyone does and pass it as a props with whatever name.